packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -356,6 +356,10 @@ export const addTag = (env: TestFixture, tag: string, props: TagProps) => {
   env.withElement(e);
 };
 
+export const addElement = (env: TestFixture, element: HTMLElement) => {
+  env.withElement(element);
+};
```

Comment:
I prefer not to add this util. Instead, you could use `addTag` or create case-specific methods like `addSmartSnippet`.

Reply:
Sure, I can create a case-specific one